packages/core/src/helpers/border-radius/border-radius.scss

```diff
@@ -1,10 +1,9 @@
 @use 'sass:list';
-
-$_allowed-sizes: ('small', 'medium', 'large', 'full');
+@use '../../integrated/tokens' as tokens;
```

Comment:
🤔  Would we be able to do `@use '@onfido/castor-tokens';` if we copied the files into `packages/tokens` instead?
That would make it part of the workspace, and we can gitignore the entire folder.
We might only need a `package.json` to name it.

Reply:
No, it's the same. On Sass you can't, we cannot predict what setup users have on their side to handle npm packages...

Reply:
Ah yea forgot about the tilde thing, nvm